setup.py

```diff
@@ -1,7 +1,8 @@
 import io
 import sys
 
-import setuptools
+from setuptools import __version__ as setuptools_version
```

Comment:
What is the connection to switching to src/ layout here? Is that some name shadowing thing?